README.md

```diff
@@ -110,3 +110,39 @@ through Django shell (`./compose run --rm api manage shell`), use the following
 >>> user.groups.add(pe_group)
 >>> user.save()
 ```
+
+## Pulp 3 Installer
+
+#### 1. Clone the GitHub repos
+
+```console
+$ git clone https://github.com/ansible/galaxy_ng.git
+$ git clone https://github.com/pulp/pulplift.git
+```
+
+#### 2. Setup pulplift
+
+```console
+$ cd pulplift
+$ git submodule update --init
+$ cp example.dev-config.yml local.dev-config.yml
+```
+
+Uncomment `pulp-ansible`, `galaxy-pulp` and `galaxy-ng` on local.dev-config.yml:
+```yaml
+pulp_install_plugins:
+  pulp-ansible:
+    source_dir: "/home/vagrant/devel/pulp_ansible"
+  galaxy-pulp:
+    source_dir: "/home/vagrant/devel/galaxy_ng/bindings/galaxy-pulp"
+  galaxy-ng:
+    source_dir: "/home/vagrant/devel/galaxy_ng"
+```
+
+#### 3. Choose a box
+
+You can find the available boxes on [pulplift](https://github.com/pulp/pulplift#available-boxes)
```

Comment:
We should really document on pulplift's README.md what the 3 different types of boxes (bare distro vs sandbox vs source) are.